Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -31,7 +60,24 @@ public override SceneQueryType SceneQueryType
         /// <remarks>
         /// This creates a dead zone in which far interaction is disabled before objects become grabbable.
         /// </remarks>
-        public float NearObjectMargin => nearObjectMargin;
+        public float NearObjectMargin
+        {
+            get => nearObjectMargin;
+            set => nearObjectMargin = value;
+        }
+
+        [SerializeField]
+        [Min(0.0f)]
+        [Tooltip("Additional distance on top of sphere cast radius when pointer is considered 'near' an object and far interaction will turn off")]
+        private float nearObjectAxisLerp = 0.9f;
+        /// <summary>
+        /// Angle range of the forward axis to query in degrees. Angle >= 360 means the entire sphere is queried
```

Comment:
`NearObjectAxisLerp` seems to me to be a lerp amount, a value between zero and 1. But this comment is referring to an angle in degrees, a value between 0 and 360. Could you please double check this comment?